data/json/itemgroups/SUS_specific_use_storage_items.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "id": "SUS_flatware",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for a flatware drawer.",
+    "subtype": "collection",
+    "items": [
+      { "item": "knife_butter", "count": [ 1, 6 ] },
+      { "item": "knife_butter", "count": [ 1, 6 ] },
```

Comment:
Wouldn't `"repeat"` be better for this?

Reply:
I don't believe it works that way in item groups?